programs/ssl/ssl_server2.c

```diff
@@ -994,7 +1031,10 @@ int main( int argc, char *argv[] )
             {
                 case 0:     opt.allow_sha1 = 0;   break;
                 case 1:     opt.allow_sha1 = 1;    break;
-                default:    goto usage;
+                default:
+                    mbedtls_printf( "Invalid value for option %s (must be 0 or 1)\n",
+                                    p );
+                    goto usage;
             }
         }
         else if( strcmp( p, "force_version" ) == 0 )
```

Comment:
Do we also print "Invalid value for option" when `force_version` is invalid?

Reply:
Yes, that's done by `mbedtls_ssl_test_parse_version`.

The auxiliary functions are not consistent, but that would be more work than I want to put in now.